examples/usage/main.tf

```
@@ -1,3 +1,34 @@
-module "terraform_module_example" {
-  source = "cloudeteer/terraform-module-example/azurerm"
+resource "azurerm_resource_group" "example" {
+  location = "germanywestcentral"
+  name     = "example-resource-group"
```

Even in examples, it's always good to follow our naming conventions.

```suggestion
  name     = "rg-example-dev-gwc-01"
```